src/gradients.rs

```diff
@@ -50,6 +50,34 @@ pub fn vertical_sobel(image: &GrayImage) -> Image<Luma<i16>> {
     filter3x3(image, &VERTICAL_SOBEL)
 }
 
+/// Scharr filter for detecting horizontal gradients.
+///
+/// Used by the [`horizontal_scharr`](fn.horizontal_scharr.html) function.
+pub static HORIZONTAL_SCHARR: [i32; 9] = [
+    3,  0,  -3,
```

Comment:
Both of these filters are in the opposite direction to the Sobel and Prewitt filters. Also, could you stick all the filters next to each other at the top of the file.